### PR TITLE
chore: make owlbot check required

### PR DESCRIPTION
**Pull request by Neenu1995**

### CODE REVIEW

**Comment by Neenu1995** [comment]
Merge after owlbot migration PR is merged.

**Review by eaball35 — APPROVED** [review]

File: .github/sync-repo-settings.yaml
Comment (kolea2) [inline .github/sync-repo-settings.yaml:19]: lgtm, but want to hold off a day or two just to make sure we didn't mess anything up with merging in the migration PR :)

**Review by kolea2 — CHANGES_REQUESTED** [review]

**Review by kolea2 — APPROVED** [review]

**Pull request merged**
